### PR TITLE
Feat/table

### PR DESCRIPTION
**Pull request by TiruDeepthi**
Please review the PR and please let me know if any changes requried.

### CODE REVIEW

File: src/api/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export const workFlowsCountUrl = "https://bc-cms-rewrite-node-ms-git-apis-raj-kumar-litmus-s-team.vercel.app/api/v1/workflows/counts"
```
Comment (RajKumar-BackCountry) [inline src/api/index.js]: here the domain name `https://bc-cms-rewrite-node-ms-git-apis-raj-kumar-litmus-s-team.vercel.app` should not be hardcoded.
Keep it in the .env file and load it here.

File: src/components/GlobalSearch.jsx [hunk 2]
```diff
@@ -3,18 +3,14 @@ import Button from './Button';
...
-      <div className='m-2'>
-          <form className="flex">   
-      <div className="w-full">
-          <input type="search" id="default-search" className="placeholder-text-bold w-full p-4 pl-10 text-sm text-gray-900 border border-borders-100 rounded bg-gray-50" placeholder="Search by Style or Title or Brand" required/>
-      </div>
-      <div className="m-0">
-        <Button className={"text-white bg-black py-3.5 px-4 rounded ml-2"}> {searchString} </Button>
-      </div>
+          <form className="flex">  
+          <div className="w-full">
```
Comment (RajKumar-BackCountry) [inline src/components/GlobalSearch.jsx]: this div is not needed. Add this class to the input tag.

File: src/components/GlobalSearch.jsx [hunk 3]
```diff
@@ -3,18 +3,14 @@ import Button from './Button';
...
+            <input type="search" className="bg-white w-full h-[64px] items-center pl-[24px] text-sm placeholder-gray-20 rounded border border-grey-30 shadow" placeholder="Search by Style or Title or Brand"/>
+          </div>
+          <div>
```
Comment (RajKumar-BackCountry) [inline src/components/GlobalSearch.jsx]: This div is also not needed

File: src/components/ManualWorkFlowDashboard.jsx [hunk 4]
```diff
@@ -5,7 +5,8 @@ function ManualWorkFlowDashboard({backButtonString, alertText, searchTest, displ
     let navigate = useNavigate();
 
     return (
-       <div className='bg-bg h-screen' >
+       <div className='bg-grey-40 h-screen' >
```
Comment (RajKumar-BackCountry) [inline src/components/ManualWorkFlowDashboard.jsx:8]: why two divs. Merge it into a single div.

File: src/components/NormalizationDashboard.jsx [hunk 5]
```diff
@@ -5,16 +5,18 @@ import WriterDashBoardTabs from "./WriterDashBoardTabs.jsx";
 
 function NormalizationDashboard() {
   return (
-    <div className="bg-bg">
-      <div className="m-2">
+    <div className="bg-grey-40 h-screen">
+      <div className="mx-[139px]">
+      <div>
```
Comment (RajKumar-BackCountry) [inline src/components/NormalizationDashboard.jsx]: Remove all unnecessary divs.

File: src/api/index.js (hunk 1 above)
Comment (RajKumar-BackCountry) [reply src/api/index.js]: I thought this file is going to have all the fetch logic.
If you just need to keep the urls.
create a folder called constants/index.js and add the urls there.

File: package.json [hunk 6]
```diff
@@ -60,6 +62,7 @@
     "jsonwebtoken": "^9.0.0",
     "jwks-rsa": "^3.0.1",
     "msal": "^1.4.18",
+    "primeicons": "^6.0.1",
```
Comment (RajKumar-BackCountry) [inline package.json]: where is this package being used ?
If it is not used, don't add it in package.json

File: package.json [hunk 7]
```diff
@@ -29,7 +29,9 @@
     "collectCoverage": true,
     "coverageDirectory": "coverage",
     "testEnvironment": "jsdom",
-    "setupFiles": ["./someModuleForTest"],
+    "setupFiles": [
+      "./someModuleForTest"
+    ],
```
Comment (RajKumar-BackCountry) [inline package.json:34]: are the unit test cases done for these changes ?

File: src/main.jsx [hunk 8]
```diff
@@ -28,7 +28,7 @@ root.render(
       <Route
         path="/normalizationDashBoard"
         element={
-          <div className="h-screen">
+          <div>
```
Comment (RajKumar-BackCountry) [inline src/main.jsx]: if you need a div just to wrap two other elements, consider using Fragments `<>`

File: package.json (hunk 6 above)
Comment (TiruDeepthi) [reply package.json]: This is needed for prime react icons

File: src/components/NormalizationDashboard.jsx (hunk 5 above)
Comment (TiruDeepthi) [reply src/components/NormalizationDashboard.jsx]: This is necessary

File: src/components/ManualWorkFlowDashboard.jsx (hunk 4 above)
Comment (TiruDeepthi) [reply src/components/ManualWorkFlowDashboard.jsx:8]: 2 div's needed as we have to give bg colors for full screen and margin for only content.

File: src/api/index.js (hunk 1 above)
Comment (TiruDeepthi) [reply src/api/index.js]: Done

File: src/components/GlobalSearch.jsx (hunk 2 above)
Comment (TiruDeepthi) [reply src/components/GlobalSearch.jsx]: done

File: src/components/GlobalSearch.jsx (hunk 3 above)
Comment (TiruDeepthi) [reply src/components/GlobalSearch.jsx]: done

File: package.json (hunk 7 above)
Comment (TiruDeepthi) [reply package.json:34]: yes, written

File: src/main.jsx (hunk 8 above)
Comment (TiruDeepthi) [reply src/main.jsx]: done

File: src/components/GlobalSearch.jsx [hunk 9]
```diff
@@ -4,12 +4,8 @@ import Button from './Button';
 function GlobalSearch({searchString}) {
     return (
           <form className="flex">  
-          <div className="w-full">
```
Comment (RajKumar-BackCountry) [inline src/components/GlobalSearch.jsx]: u removed `w-full` but you didnt add it to input.
Did u test the changes ?

File: src/constants/index.js [hunk 10]
```diff
@@ -0,0 +1,3 @@
+const { VITE_LOCAL_URL: LOCAL_URL } = process.env;
+
+export const workFlowsCountUrl = `${LOCAL_URL}/workflows/counts`
```
Comment (RajKumar-BackCountry) [inline src/constants/index.js]: It's not LOCAL_URL.
Try HOST_NAME.

File: src/components/NormalizationDashboard.jsx (hunk 5 above)
Comment (RajKumar-BackCountry) [reply src/components/NormalizationDashboard.jsx]: Necessary how ?

File: package.json (hunk 6 above)
Comment (RajKumar-BackCountry) [reply package.json]: Which prime react icon are u using and why ?
Ideally all the icons should come from Arun.

File: src/components/GlobalSearch.jsx (hunk 9 above)
Comment (TiruDeepthi) [reply src/components/GlobalSearch.jsx]: yes i tested

File: src/components/StatusBar.jsx [hunk 11]
```diff
@@ -1,15 +1,15 @@
...
-        <p className="text-sm mr-8">{title}</p>
-        <h2 className="font-bold text-xl">{count}</h2>
-      </div>
+    <div className="bg-white rounded-lg shadow flex flex-row w-[260px] h-[123px] items-center pl-[25px] mt-[64px]">
+    <div className={`${className} flex justify-center items-center rounded-full w-[50px] h-[50px] `}>
+        <img className={"w-[18px] h-[18px]"} src={img} alt="Assigned SVG" />
     </div>
+    <div className="m-3 mx-8">
+        <p className="text-sm">{title}</p>
+        <h2 className="font-bold text-[34px]">{`${(count)/100}k`}</h2>
```
Comment (RajKumar-BackCountry) [inline src/components/StatusBar.jsx]: It's not 100.
It's 1000.

Comment (TiruDeepthi) [reply src/components/StatusBar.jsx]: done

File: src/constants/index.js (hunk 10 above)
Comment (TiruDeepthi) [reply src/constants/index.js]: done

File: src/components/NormalizationDashboard.jsx (hunk 5 above)
Comment (TiruDeepthi) [reply src/components/NormalizationDashboard.jsx]: we need bg color for full screen but spacing only for content if i move it to single div the bg color will be only for the content

File: package.json (hunk 6 above)
Comment (TiruDeepthi) [reply package.json]: ok then we can remove

File: src/components/GlobalSearch.jsx [hunk 12]
```diff
@@ -1,20 +1,12 @@
...
-      </div>
-      <div className="m-0">
-        <Button className={"text-white bg-black py-3.5 px-4 rounded ml-2"}> {searchString} </Button>
-      </div>
-  </form>
-      </div>
-      </div>
+          <form className="flex">  
+            <input type="search" className={className} placeholder="Search by Style or Title or Brand"/>
+            <Button className={"text-white bg-black h-[64px] w-[131px] rounded ml-2"}> {searchString} </Button>
```
Comment (RajKumar-BackCountry) [inline src/components/GlobalSearch.jsx]: I can still see hardcoded classNames.

File: src/components/StatusBarsForNormalization.jsx [hunk 13]
```diff
@@ -1,41 +1,53 @@
...
 import InProgress from "../logos/InProgress.svg";
 import Completed from "../logos/Completed.svg";
 import StatusBar from "./StatusBar";
+import {workFlowsCountUrl} from '../constants/index'
 
 function StatusBarsForNormalization() {
+  const [statusBarData, setStatusBarData] = useState([]);
+
+ useEffect(() => {
+        fetch(workFlowsCountUrl,{
```
Comment (RajKumar-BackCountry) [inline src/components/StatusBarsForNormalization.jsx:13]: While you are fetching this, are you showing any loaders in the statusbar component ?

File: src/components/GlobalSearch.jsx (hunk 12 above)
Comment (TiruDeepthi) [reply src/components/GlobalSearch.jsx]: Button is a component we are sending className as prop to Button component.

File: src/components/StatusBarsForNormalization.jsx (hunk 13 above)
Comment (TiruDeepthi) [reply src/components/StatusBarsForNormalization.jsx:13]: no we don't

File: src/components/GlobalSearch.jsx (hunk 12 above)
Comment (RajKumar-BackCountry) [reply src/components/GlobalSearch.jsx]: In globalSearch create two props. 
inputClasses, buttonClasses and pass it.
Dont hardcode.

Comment (TiruDeepthi) [reply src/components/GlobalSearch.jsx]: done with the change

File: src/components/NormalizationDashboard.jsx [hunk 14]
```diff
@@ -9,7 +9,8 @@ function NormalizationDashboard() {
       <div className="mx-[139px]">
         <StatusBarsForNormalization />
       <div className="mt-[40px]">
-        <GlobalSearch searchString={"Search"} className="bg-white w-full h-[64px] items-center pl-[24px] text-sm placeholder-gray-20 rounded border border-grey-30 shadow" />
+        <GlobalSearch searchString={"Search"} inputClasses="bg-white w-full h-[64px] items-center pl-[24px] text-sm placeholder-gray-20 rounded border border-grey-30 shadow"
```
Comment (RajKumar-BackCountry) [inline src/components/NormalizationDashboard.jsx]: Either enclose the classes in {} or dont.
Pick one standard and follow it through out the repo.
Dont mix and match both.

Comment (TiruDeepthi) [reply src/components/NormalizationDashboard.jsx]: done

**Pull request merged**